planar/triangulate/triangulate.go

```diff
@@ -0,0 +1,16 @@
+package triangulate
+
+import "github.com/go-spatial/geom"
+
+type Interface interface {
+	// SetPoints sets the nodes to be used in the triangulation
+	SetPoints(pts []geom.Point, data []interface{})
```

Comment:
what is `data`?

Reply:
Should SetPoints overwrite all existing points? If not, then I would call it `AddPoints`.

Reply:
How can we access the data? Via geom.Triangle? Is the same data attached to all vertices that are build from pts?

Reply:
This is a good point. Maybe Triangles should return a [][3]interface{} for the data associated with each triangle? The more I think about this, the more I think we should drop the metadata, and allow the user to manage that outside of this interface.